scripts/unmount_unmodified_studies.py

```
@@ -0,0 +1,101 @@
+
+    if not studies:
+        print("No unmodified studies found.")
+        return
+
+    print(f"Found {len(studies)} unmodified study/studies.")
+
+    if batch_size is not None and batch_size < len(studies):
+        print(f"Batch size limit applied: processing {batch_size} out of {len(studies)} studies.")
+        studies = studies[:batch_size]
+
+    print(f"Studies to process:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)